GameServer/GameServer/Player.cs

```diff
@@ -169,8 +173,7 @@ public void AddItem(string _type, int _count)
         }
         public void Damage(int _healthDelta, int _animalId)
         {
-
-            health -= _healthDelta;
+            health -= (int)(_healthDelta * GameLogic.armor[this.armor]); // redeucing damage based on armor type
```

Comment:
Can you just call changehealth here so that the calculation only happens in one place

Reply:
That's a good idea, implementing it now.